<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>screensharing_window</class>
 <widget class="QMainWindow" name="screensharing_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blink Screen Sharing</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/blink48.png</normaloff>icons/blink48.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="window_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="toolbar" native="true">
      <layout class="QHBoxLayout" name="toolbar_layout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="color_depth_button">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Higher color depths provide better quality but also require more bandwidth. Lowering it may help if screen sharing feels sluggish or it is lagging.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Default Color Depth</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>True Color (24 bits)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>High Color (16 bits)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Low Color (8 bits)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="toolbar_spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>15</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="scale_button">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Scale</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/scale.svg</normaloff>icons/scale.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="viewonly_button">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>View only</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/viewonly.svg</normaloff>icons/viewonly.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="screenshot_button">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Screenshot</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/screenshot.svg</normaloff>icons/screenshot.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="fullscreen_button">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Fullscreen</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/fullscreen.svg</normaloff>
           <normalon>icons/fullscreen-exit.svg</normalon>
           <disabledon>icons/fullscreen-exit.svg</disabledon>
           <activeon>icons/fullscreen-exit.svg</activeon>
           <selectedon>icons/fullscreen-exit.svg</selectedon>icons/fullscreen.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scroll_area">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="vncviewer">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>7</y>
         <width>987</width>
         <height>611</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
